lab_3_keywords_textrank/main.py

```diff
@@ -215,7 +238,10 @@ def __init__(self) -> None:
         """
         Constructs all the necessary attributes for the adjacency matrix graph object
         """
-        pass
+        self._matrix = []
+        self._vertices = []
+        self._positions = {}
+        self._position_weights = {}
 
     # Step 4.2
     def add_edge(self, vertex1: int, vertex2: int) -> int:
```

Comment:
we've discussed this approach with colleagues and we came to the conclusion that although this approach can be functional it is advised to only use matrix to store 0 or 1, not the actual vertices 

think of a matrix in math: it does not contain names for the rows and columns, it is purely values
please, try to come up with a way to store information about position-vertex links somehow else 

possibly, i'll write a post about it in the channel... as it is a common issue

Reply:
but i put vertices into a separate list, wouldn't that solve the problem?

Reply:
yep I see I does